### PR TITLE
Time feature

### PR DESCRIPTION
**Pull request by warrermind**
Pull request: TimeFeature ready for review

### CODE REVIEW

File: Microwave.App/Program.cs [hunk 1]
```diff
@@ -30,7 +31,7 @@ static void Main(string[] args)
 
             CookController cooker = new CookController(timer, display, powerTube);
 
-            UserInterface ui = new UserInterface(powerButton, timeButton, startCancelButton, door, display, light, buzzer, cooker);
+            UserInterface ui = new UserInterface(powerButton, timeButton, startCancelButton, subtractTimeButton, door, display, light, cooker);
```
Comment (JAY-Z-Sharp) [inline Microwave.App/Program.cs]: BuzzerFeature still needs to be a parameter in the UserInterface constructor.

File: Microwave.App/Program.cs [hunk 2]
```diff
@@ -30,7 +31,7 @@ static void Main(string[] args)
 
             CookController cooker = new CookController(timer, display, powerTube);
 
-            UserInterface ui = new UserInterface(powerButton, timeButton, startCancelButton, door, display, light, buzzer, cooker);
```
Comment (FrederikLaursenSW) [inline Microwave.App/Program.cs:33]: Notice that buzzer has been removed, and might need to be added again

**Review by warrermind — COMMENTED** [review]
Reviewed myself. Everything looks good. Notice that i have added changes to the integrations-tests. Maybe we should deactive that project completely in master now?

File: Microwave.Classes/Controllers/UserInterface.cs [hunk 3]
```diff
@@ -79,6 +81,19 @@ public void OnTimePressed(object sender, EventArgs e)
...
+                case States.COOKING:
+                    myCooker.AddCookingTime();
+                    break;
+            }
+        }
+
+        public void OnSubtractTimePressed(object sender, EventArgs e)
+        {
+            switch (myState)
+            {
```
Comment (FrederikLaursenSW) [inline Microwave.Classes/Controllers/UserInterface.cs:93]: A defined default behavior would be nice

File: Microwave.Classes/Controllers/UserInterface.cs [hunk 4]
```diff
@@ -79,6 +81,19 @@ public void OnTimePressed(object sender, EventArgs e)
...
+                case States.COOKING:
```
Comment (FrederikLaursenSW) [inline Microwave.Classes/Controllers/UserInterface.cs:94]: Defining a default behavior when not in a cooking state would help on readability.

**Review by JAY-Z-Sharp — CHANGES_REQUESTED** [review]

File: Microwave.App/Program.cs (hunk 2 above)
Comment (warrermind) [reply Microwave.App/Program.cs:33]: Has been changed now.

File: Microwave.App/Program.cs (hunk 1 above)
Comment (warrermind) [reply Microwave.App/Program.cs]: Has been added.

**Review by JAY-Z-Sharp — APPROVED** [review]

File: Microwave.Classes/Controllers/UserInterface.cs (hunk 4 above)
Comment (warrermind) [reply Microwave.Classes/Controllers/UserInterface.cs:94]: Agree. But following code standard of existing code, that has no default state.

**Review by FrederikLaursenSW — APPROVED** [review]
The has been reviewed and is approved for merge

**Pull request merged**